### PR TITLE
iterx: paging iteration working

### PR DESCRIPTION
**Pull request by dahankzter**
We used to rely upon NumRows to determine if there are
new pages available. This is not correct and the needed
data to do this is not exposed by the gocql driver.

We simply remove these checks and let the driver decide.

### CODE REVIEW

File: iterx.go [hunk 1]
```diff
@@ -180,7 +170,13 @@ func (iter *Iterx) scanAll(dest interface{}, structOnly bool) bool {
 
 		// allocate memory for the page data
 		if !alloc {
-			v = reflect.MakeSlice(slice, 0, iter.Iter.NumRows())
+			sz := iter.NumRows()
+			if sz == 0 {
+				// Taken out of the air.
+				// Should it be the default page size?
+				sz = 100
```
Comment (mmatczuk) [inline iterx.go]: Note that users may change that https://godoc.org/github.com/gocql/gocql#Query.PageSize so it should be a default unless changed.

Comment (mmatczuk) [reply iterx.go]: Not sure exactly why NumRows is wrong, maybe it's an gocql error?

File: iterx_test.go [hunk 2]
```diff
@@ -327,3 +328,46 @@ func TestNotFound(t *testing.T) {
...
+		defer it.Close()
+		var cnt int
+		for {
+			p := &Paging{}
+			if !it.StructScan(p) {
+				break
+			}
+			cnt++
+		}
+		if cnt != 100 {
```
Comment (mmatczuk) [inline iterx_test.go]: If default page size is 5k do we ever hit the bug without the patch here.

File: .travis_start_db.sh [hunk 3]
```diff
@@ -4,7 +4,7 @@ set -e
 case ${DB} in
 
 scylla)
-    sudo curl -o /etc/apt/sources.list.d/scylla.list -L http://repositories.scylladb.com/scylla/repo/20fc70b18261bf832cf8e0733a27979c/ubuntu/scylladb-2.1-trusty.list
+    sudo curl -o /etc/apt/sources.list.d/scylla.list -L http://repositories.scylladb.com/scylla/repo/20fc70b18261bf832cf8e0733a27979c/ubuntu/scylladb-3.0-trusty.list
```
Comment (mmatczuk) [inline .travis_start_db.sh]: We can piggyback that it could go as a separate PR.

File: iterx_test.go (hunk 2 above)
Comment (dahankzter) [reply iterx_test.go]: I can't say for sure but it seems to vary with the workload and data set size.

File: .travis_start_db.sh (hunk 3 above)
Comment (dahankzter) [reply .travis_start_db.sh]: Ok. Putting up another PR.
